presidio-analyzer/presidio_analyzer/predefined_recognizers/phone_recognizer.py

```diff
@@ -66,52 +60,36 @@ def analyze(
         :return: List of phone numbers RecognizerResults
         """
         results = []
-        for entity in entities:
-            region = entity.replace(ENTITY_TYPE_SUFFIX, "")
-            if region in SUPPORTED_REGIONS or entity is INTERNATIONAL_ENTITY_TYPE:
-                for match in phonenumbers.PhoneNumberMatcher(text, region, leniency=0):
-                    international_phone_prefix = match.raw_string.startswith("+")
-                    if entity == INTERNATIONAL_ENTITY_TYPE \
-                            and international_phone_prefix:
-                        results += [self._get_international_recognizer_result(match)]
-                    # phone-numbers matches international numbers twice
-                    elif not international_phone_prefix:
-                        results += [
-                            self._get_regional_recognizer_result(
-                                match, entity, text, nlp_artifacts
-                            )
-                        ]
-
-        return results
-
-    def _get_regional_recognizer_result(self, match, entity, text, nlp_artifacts):
+        for region in self.supported_regions:
+            for match in phonenumbers.PhoneNumberMatcher(text, region, leniency=1):
```

Comment:
Raised leniency since phone numbers matched '1234567', credit cards and us driving numbers. 
After raising, only some regions matched '1234567', which I excluded above (see LOW_LENIENCY_CODES)

Reply:
This could have an impact with false positives. Based on your tests, was that the case? Maybe worth checking more than just 1234567 (e.g. take a few samples from a dataset). Theoretically we could keep all the changes in this PR, but still stick with the five regions we had in the first place + documentation on how to add more, right?

Reply:
The purpose of raising the leniency was to avoid FPs. When it was lower, lots of regions matched random numbers, credit cards, licence numbers. Having the fixed regions will help with reducing those FPs so I might revert back to using them after all, at least as a first step.